micro-rdk-ffi/src/ffi/runtime.rs

```diff
@@ -257,7 +288,7 @@ pub unsafe extern "C" fn viam_server_start(ctx: *mut viam_server_context) -> via
         use micro_rdk::common::credentials_storage::RAMStorage;
         use micro_rdk::common::credentials_storage::RobotConfigurationStorage;
         use micro_rdk::proto::provisioning::v1::CloudConfig;
-
+        log::info!("when the FFI was built a robot config was supplied, defaulting to this robot");
```

Comment:
Maybe take this out for now but add a `TODO(RSDK-9715)` which is my init logging ticket so we can do this in coordination with other startup logging changes?